mysql-test/suite/rocksdb/t/use_direct_reads_writes.test

```diff
@@ -1,47 +1,61 @@
 --source include/have_rocksdb.inc
 
+call mtr.add_suppression("RocksDB: Can't enable both");
```

Comment:
The suppressions should be redundant now as they are not making it to the main server error log